README.md

```diff
@@ -196,7 +196,7 @@ A collection of android security related resources.
 3. [android app security checklist](https://github.com/b-mueller/android_app_security_checklist)
 4. [Mobile App Pentest Cheat Sheet](https://github.com/tanprathan/MobileApp-Pentest-Cheatsheet)
 5. ~~[Mobile Security Reading Room](https://mobile-security.zeef.com) - A reading room which contains well categorised technical reading material about mobile penetration testing, mobile malware, mobile forensics and all kind of mobile security related topics~~
-6. ~~[Android Reverse Engineering 101 by Daniele Altomare](http://www.fasteque.com/android-reverse-engineering-101-part-1/)~~
+6. [Android Reverse Engineering 101 by Daniele Altomare](http://web.archive.org/web/20180721134044/http://www.fasteque.com:80/android-reverse-engineering-101-part-1/)
```

Comment:
Can you change `[Android Reverse Engineering 101 by Daniele Altomare]` to `[Android Reverse Engineering 101 by Daniele Altomare - web archive link]`?